src/pages-and-resources/data/slice.js

```diff
@@ -19,13 +20,17 @@ const slice = createSlice({
     updateSavingStatus: (state, { payload }) => {
       state.savingStatus = payload.status;
     },
+    updateCourseAppsApiStatus: (state, { payload }) => {
+      state.CourseAppsApiStatus = payload;
```

Comment:
can we just store `status` here and not the complete payload object?
e.g.`payload.status;`